test/splitter.js

```diff
@@ -0,0 +1,206 @@
+const truffleAssert = require('truffle-assertions');
+const Splitter = artifacts.require("./Splitter.sol");
+
+contract('Splitter', async accounts => {
+
+    const { toBN, toWei } = web3.utils;
+
+    const getGasCost = async txObj => {
+        const gasUsed = txObj.receipt.gasUsed;
+        const tx = await web3.eth.getTransaction(txObj.tx);
+        const gasPrice = tx.gasPrice;
+
+        return toBN(gasUsed).mul(toBN(gasPrice));
+    };
+
+    const checkEventNotEmitted = async () => {
+        const result = await truffleAssert.createTransactionResult(splitter, splitter.transactionHash);
+
+        await truffleAssert.eventNotEmitted(
+            result
+        );
+    }
+
+    const [ contractOwnerAddress, sender, recipientOne, recipientTwo] = accounts;
+    let splitter;
+
+    beforeEach("deploy and prepare", async function() {
+        splitter = await Splitter.new({from: contractOwnerAddress});
+    });
+
+    it('Sender sends 5 wei, 4 wei is split equally between recipients and 1 wei sent to the senders balance', async () => {
+
+        // Take snapshot of initial balances
+        const initContractEthBalance = toBN(await web3.eth.getBalance(splitter.address));
```

Comment:
I dearly hope this value starts at 0.
In which case, on line 45, it can be `, "5"`.